README.md

```diff
@@ -1,6 +1,6 @@
 # create-unirep-app
 
-This is a demo app of a [unirep](https://github.com/Unirep/Unirep) attester. In this demo app, users can request reputation from the example attester. After transition, user can prove how much reputation he has.
+This is a demo app of a [unirep](https://github.com/Unirep/Unirep) attester. In this demo app, users can request data from the example attester. After transition, user can prove how much data he has.
```

Comment:
I'd recommend taking the `yarn start` command out of the readme. The script does not work when there are errors and will cause the user machine to end up in a weird state.